src/components/pages/Login/_SignupPageStyled.less

```diff
@@ -5,40 +5,121 @@
 @font: 'Lato', sans-serif;
 
 .signup-root {
-  width: 100%;
+  width: 100vw;
+  height: 100vh;
+  overflow:hidden;
+  position: relative;
   display: flex;
-  background-image: url('../../../styles/blueStatueOfLiberty.png');
-  background-size: cover;
-  background-repeat: no-repeat;
+  justify-content: center;
+  align-items: center; 
+}
+
+.background{
+  width:100%;
+  height:100vh;
+  z-index:-3;
+  position: relative;
+}
+
+.svgContain{
```

Comment:
same here, spacing before the `{` is inconsistent with other property declarations.